src/main/java/frc/robot/commands/CmdManipulatorSetIntakeSpeed.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Manipulator;
+
+public class CmdManipulatorSetIntakeSpeed extends CommandBase 
```

Comment:
We should split this command into two separate commands: one for intaking a game piece, and one for releasing it. The two actions are fundamentally different.

This command will intake a game piece and stop when the intake sensor senses a game piece.

The other command will reverse the motors and stop after some short period of time.